docs/src/main/tut/contravariant.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title:  "Contravariant"
+section: "typeclasses"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/functor/Contravariant.scala"
```

Comment:
This should switch from `non` to `typelevel`. We probably have the same mistake in lots of docs. We could probably just give relative paths here and use site-wide config for the base source URL.